app/helpers/stats_helper.rb

```
@@ -0,0 +1,233 @@
+
+    distance_value =
+      if distance_unit == 'mi'
+        (peak[1] / 1609.34).round(2)
+      else
+        distance_km
+      end
+
+    text = "#{date.strftime('%B %d')} (#{distance_value} #{distance_unit})"
+
+    link_to text, map_url(start_at: date.beginning_of_day, end_at: date.end_of_day), class: 'underline'
+  end
```

RuboCop (1.76.1)

[convention] Assignment Branch Condition size for `peak_day` is too high. [<6, 23, 5> 24.29/17]

(Metrics/AbcSize)